pkg/controller/controller.go

```diff
@@ -190,6 +192,86 @@ func (c *Controller) initPodServiceAccount() {
 	// actual service accounts are deployed at the time of Postgres/Spilo cluster creation
 }
 
+func (c *Controller) initRole() {
+
+	// service account on its own lacks any rights starting with k8s v1.8
+	// operator binds it to the namespaced role with sufficient privileges
+	if c.opConfig.PodServiceAccountRoleDefinition == "" {
+		c.opConfig.PodServiceAccountRoleDefinition = fmt.Sprintf(`
+		{
```

Comment:
Is it possible to not store this in the code file, maybe just as some sort of a resource?

Reply:
We should make this a seperate issue, to see where we have ugly text in the Go code to be replaced by code or templates.

Reply:
I've checked, and aside from tests this is the only place where it's used. I agree, we should do it in either go code or files, but I thought the code addition would be easier to review if it comes in the same style as the `ServiceAccountDefinition` and `ServiceAccountRoleBindingDefinition`.